h2o-docs/src/product/data-science/hglm.rst

```diff
@@ -0,0 +1,407 @@
+Hierarchical Generalized Linear Model (HGLM) 
+============================================
+
+Introduction
+------------
+
+Hierarchical linear models (HLM) are used in situations where measurements are taken with clusters of data and there are effects of the cluster that can affect the coefficient values of GLM. 
+
+For instance, if we measure the performance of students from multiple schools along with other predictors like family annual incomes, students' health, school type (public, private, religious, etc.), and etc., we would suspect that students from the same school will have similar performances than students from different schools. Therefore, we can denote a coefficient for predictor :math:`m \text{ as } \beta_{mj}` where :math:`j` denotes the school index in our example. :math:`\beta_{0j}` denotes the intercept associated with school :math:`j`.
+
+A level-1 HLM can be expressed as:
+
+.. math::
+   
+   y_{ij} = \beta_{0j} + \sum_{m=1}^{p-1} x_{mij} \beta{mj} + \varepsilon_{ij} \quad \text{ equation 1}
+
+The level-2 model can be expressed as:
+   
+.. math::
+   
+   \beta_{0j} = \beta_{00} + u_{0j}, \beta_{mj} = \beta_{m0} + u_{mj} \quad \text{ equation 2}
+
+where:
+
+- :math:`j(=[1,2,...,J])` denotes the cluster (level-2 variable) the measurement is taken from (e.g. the school index);
+- :math:`i(=1,2,...,n_j)` denotes the data index taken from within cluster :math:`j`;
+- :math:`\beta_{00}` is the fixed intercept;
+- :math:`\beta_{0j}` is the random intercept;
+- :math:`\beta_{m0}` is the fixed coefficient for predictor :math:`m`;
+- The dimension of fixed effect coefficients is :math:`p` which includes the intercept;
+- :math:`u_{mj}` is the random coefficient for predictor :math:`m`. For predictors without a random coefficient, :math:`u_{mj} = 0`;
+- The dimension of the random effect coefficients is :math:`q` which can include the intercept. Note that :math:`q \leq p`;
+- :math:`\varepsilon_{ij} \sim N(0, \delta_e^2)`;
+- :math:`u_{ij} \sim N(0, \delta_u^2)`:
+- :math:`\varepsilon_{ij}, u_{mj}` are independent;
+- :math:`u_{mj}, u_{m,j}` are independent if :math:`m \neq m`.
+
+We need to solve the following parameters: :math:`\beta_{00}, \beta_{0j}, \beta_{m0}, u_{mj}, \delta_e^2, \delta_u^2`. To do this, we use the standard linear mixed model expressed with vectors and matrices:
```

Comment:
remove the following starting from 

To do this, ....

and stop at 

Defining an HGLM model